### PR TITLE
New Aron review

### PR DESCRIPTION
**Pull request by desmax74**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -250,7 +253,7 @@ Support tickets for RedHat Trusted Profile Analyzer can be opened at https://acc
 
 ## Release notes and Roadmap
 
-Release notes can be found [here](https://docs.redhat.com/en/documentation/red_hat_trusted_profile_analyzer/1.2/html/release_notes/index).
+Release notes can be found [here](https://access.redhat.com/products/red-hat-trusted-profile-analyzer).
```
Comment (ritz303) [inline README.md]: Please revert this change.  The URL that was previously there was correct.

Comment (ritz303) [reply README.md]: This URL (https://access.redhat.com/products/red-hat-trusted-profile-analyzer) is to replace the URL on line 260 under the "Related Information" section.

Comment (desmax74) [reply README.md]: ok

File: README.md [hunk 2]
```diff
@@ -213,20 +214,22 @@ collections:
...
+If you install any collection from Ansible Galaxy, upgrading the Ansible package is not automatically done.
 To upgrade the collection to the latest available version, run the following command:
 
 ```
 ansible-galaxy collection install redhat.trusted_profile_analyzer --upgrade
 ```
 
-You can also install a specific version of the collection, for example, if you need to downgrade when something is broken in the latest version (please report an issue in this repository). Use the following syntax to install version 0.2.0:
+You can also install a specific version of the collection.
+You can also install a specific version of the collection. (please report an issue in this repository). Use the following syntax to install version 0.2.0:
```
Comment (ritz303) [inline README.md]: Please remove this line, it's just a duplicate of 233, with the added parenthetical statement at the end, which we don't want in there. Replace the line with:

`For example, if you need to downgrade when something is broken in the latest version.`

Comment (ritz303) [reply README.md]: It should look like this:

```
You can also install a specific version of the collection.
For example, if you need to downgrade when something is broken in the latest version.
```

Comment (desmax74) [reply README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -213,20 +216,22 @@ collections:
   - name: redhat.trusted_profile_analyzer
 ```
 
-or with the classic ansible command
+Or by using the following Ansible commands:
 
 ```shell
 ANSIBLE_ROLES_PATH="roles/" ansible-playbook -i inventory.ini play.yml -vv
```
Comment (ritz303) [inline README.md]: Is this a command a user can copy and paste as-is? Does the ANSIBLE_ROLES_PATH variable have to be exported first? Should it be like this?

export ANSIBLE_ROLES_PATH="roles/" ; ansible-playbook -i inventory.ini play.yml -vv

Comment (desmax74) [reply README.md]: ok
